src/scss/_mixins.scss

```diff
@@ -80,11 +80,15 @@
 		display: block;
 	}
 
-	// Standardise outline styles
-	:focus {
-		outline: 2px solid $o-normalise-focus-color;
+	// Standardise focus styles. This is scoped because IE allows the body tag to have focus which is... odd
+	a,
+	button,
+	[tabindex]{
```

Comment:
Have we thought about applying this default focus style to input and textarea nodes?

Reply:
like we do on line 92?